igcollect/logfile_values.py

```diff
@@ -276,12 +293,17 @@ def main():  # NOQA: C901
                         for line in fh:
```

Comment:
I know it is not your code and might be out of scope .. but, we should get rid of that indention mess here.

Reply:
it`s my code btw))

Reply:
Then maybe you can tackle this thing during the next PR or when you have some time do some refactoring for it. 😉